### PR TITLE
Add labelProtection

### PR DESCRIPTION
**Pull request by miroslawtrzupek**

### CODE REVIEW

**Comment by miroslawtrzupek** [comment]
comment

**Comment by miroslawtrzupek** [comment]
comment

**Comment by mtzstiolimanowa** [comment]
good

File: WpfGit/MainWindow.xaml [hunk 1]
```diff
@@ -12,5 +12,6 @@
         <Button Content="feature-branch"/>
         <Button Content="button-branch"/>
         <Button Content="label-branch"/>
```
Comment (mtzstiolimanowa) [inline WpfGit/MainWindow.xaml:14]: ok

**Review by mtzstiolimanowa — COMMENTED** [review]
ok

**Review by mtzstiolimanowa — APPROVED** [review]
fa

**Pull request merged**